### PR TITLE
Add test to verify excess rules and meters deletion while sync

### PR DESCRIPTION
**Pull request by ylobankov**

### CODE REVIEW

File: services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceImpl.java [hunk 1]
```diff
@@ -255,6 +257,14 @@ public RulesSyncResult synchronizeSwitchRules(SwitchId switchId) {
                 new HttpEntity(buildHeadersWithCorrelationId()), RulesSyncResult.class, switchId).getBody();
     }
 
+    @Override
+    public SwitchSyncResult synchronizeSwitch(SwitchId switchId, boolean removeExcess) {
```
Comment (rtretyak) [inline services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceImpl.java]: shouldn't there be separate flags for excess rules and excess meters?

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+            [sw.dpId, northbound.getAllMeters(sw.dpId).meterEntries.findAll {
+                it.meterId > MAX_SYSTEM_RULE_METER_ID
+            }*.meterId]
+        }
+
+        involvedSwitches.each { northbound.deleteSwitchRules(it.dpId, DeleteRulesAction.IGNORE_DEFAULTS) }
+        [srcSwitch, dstSwitch].each { northbound.deleteMeter(it.dpId, metersMap[it.dpId][0]) }
+        Wrappers.wait(RULES_DELETION_TIME) {
+            involvedSwitches.each { assert northbound.switchValidate(it.dpId).rules.missing.size() == 2 }
+            [srcSwitch, dstSwitch].each { assert northbound.switchValidate(it.dpId).meters.missing.size() == 1 }
```
[the remark this answers is not in the document]
Comment (ylobankov) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy]: Will do.

File: services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceImpl.java (hunk 1 above)
Comment (ylobankov) [reply services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceImpl.java]: For now, API doesn't support separate flags for rules and meters. So we need to ask here #2189 about this. I will do it.

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy (hunk 2 above)
[the remark this answers is not in the document]
Comment (ylobankov) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy]: done

**Pull request merged**